mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule RailwayIpc.MixProject do
   def project do
     [
       app: :railway_ipc,
-      version: "0.2.6",
+      version: "0.2.7",
```

Comment:
This is the version update for Railway, right?

Reply:
Yes